extensions/roc-package-webpack-node-dev/src/config/roc.config.js

```diff
@@ -4,6 +4,8 @@ export default {
             watch: [
                 'roc.config.js',
             ],
+            inspect: false,
```

Comment:
I think this is a good idea, especially since we have a `--dev-debug` already that can be confused with `--dev-debug-brk`.